Analytics/Classes/Internal/SEGUserDefaultsStorage.h

```diff
@@ -0,0 +1,20 @@
+//
+//  SEGUserDefaultsStorage.h
+//  Analytics
+//
+//  Created by Tony Xiao on 8/24/16.
+//  Copyright © 2016 Segment. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "SEGStorage.h"
+
+@interface SEGUserDefaultsStorage : NSObject <SEGStorage>
+
+@property (nonatomic, strong, nullable) id<SEGCrypto> crypto;
```

Comment:
Hmmm...  I see. I think that's ok, the user defaults code is only for tvOS and is only been available for a single release, and it gives us a window to change some of the implementation and we should use it now rather than be stuck with a legacy implementation.